app/services/instrumentation.rb

```diff
@@ -0,0 +1,34 @@
+module Instrumentation
+  class << self
+    def append_to_log(args = {})
+      RequestStore.store[:custom_log_items] ||= {}
+      RequestStore.store[:custom_log_items].merge!(args) unless args.blank?
+      RequestStore.store[:custom_log_items]
+    end
+    alias_method :custom_log_items, :append_to_log
```

Comment:
Ah, I think my use of `args` here is confusing.  it is actually the logged payload.  The `custom_log_items` alias is simply for clarity when actually logging the data.